src/Login.ts

```diff
@@ -67,6 +70,11 @@ interface ILoginParams {
 }
 /* eslint-enable camelcase */
 
+export const DELEGATED_OIDC_COMPATIBILITY = new UnstableValue(
+    "delegated_oidc_compatibility",
+    "org.matrix.msc3824.delegated_oidc_compatibility",
+);
```

Comment:
I think this should be in the js-sdk

Reply:
It can be in js-sdk but only after https://github.com/matrix-org/matrix-js-sdk/pull/2633 is merged (and probably https://github.com/matrix-org/matrix-react-sdk/pull/9232)

Reply:
I'm going to make this PR draft until https://github.com/matrix-org/matrix-react-sdk/pull/9232 is merged.